samples/net/coap_client/src/coap-client.c

```diff
@@ -196,6 +199,26 @@ static void event_iface_up(struct net_mgmt_event_callback *cb,
 	}
 
 	k_delayed_work_init(&retransmit_work, retransmit_request);
+}
+
+static void event_iface_up(struct net_mgmt_event_callback *cb,
```

Comment:
this is no longer a callback function registered and called when an interface is up. please update the interface to remove the unnecessary parameters and change the name to avoid misleading

Reply:
I have to keep break this function into two then only I can test other methods at a same time. Otherwise it is not possible.

	event_iface_init();
	printk("\n ************ COAP Client  test 1 *************\n");
	event_iface_up(NULL, NET_EVENT_IF_UP, iface, COAP_METHOD_GET);

So that I have implemented. if you want to give new name to event_iface_init(); function let me know.